src/test/java/net/snowflake/client/jdbc/ConnectionLatestIT.java

```diff
@@ -64,6 +67,57 @@ public void tearDown() throws InterruptedException {
     service.resetNumOfRetryToTriggerTelemetry();
   }
 
+  /**
+   * Verify the passed heartbeat frequency matches the output value if the input is valid (between
+   * 900 and 3600).
+   */
+  @Test
```

Comment:
Previously, this test tested that the connection parameter was set. Now, it tests that the connection parameter is set AND the SFSession variable corresponding to the connection parameter is also set.